common/socks5/requests/src/lib.rs

```diff
@@ -32,6 +32,11 @@ pub enum Socks5RequestError {
 
     #[error(transparent)]
     ProviderInterfaceError(#[from] ServiceProviderMessagingError),
+
+    #[error("received unsupported request protocol version: {protocol_version:?}")]
```

Comment:
I think `ProtocolVersion` has a `Display`, no? (unless I'm misremembering)

Reply:
ah yes, it has